src/Microsoft.TemplateEngine.Cli/LocalizableStrings.resx

```diff
@@ -434,4 +434,36 @@
     <value>{0} {1}
     The default value '{1}' is not a valid value for {0}.</value>
   </data>
+  <data name="AddRefPostActionFailed" xml:space="preserve">
+    <value>Failed to add reference {0} to project file {1}</value>
+  </data>
+  <data name="AddRefPostActionMultipleProjFiles" xml:space="preserve">
+    <value>Found multiple possible project files in the same directory. Could not determine which to add a reference to. Files:</value>
```

Comment:
It might be clearer in the full context of its usage:
![addrefmultiprojfiles](https://cloud.githubusercontent.com/assets/20912881/24875528/35a529e0-1ddd-11e7-86ea-1506bd9feb06.PNG)